homeassistant/components/squeezebox/config_flow.py

```diff
@@ -222,3 +242,55 @@ async def async_step_dhcp(
 
         # if the player is unknown, then we likely need to configure its server
         return await self.async_step_user()
+
+
+class OptionsFlowHandler(OptionsFlow):
+    """Options Flow Handler."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
```

Comment:
This would not be necessary if https://github.com/home-assistant/core/pull/129562 was merged...

Reply:
OK, but that's not merged yet I think, so not sure what you're saying.  Could you explain?

Reply:
:-)

Reply:
It's been merged... so you can drop the init (and not pass the arg)